test/services/json_to_events_test.rb

```diff
@@ -5,15 +5,15 @@ class JsonToEventsTest < ActiveSupport::TestCase
     @raw_json = File.open(Rails.root.join("test/fixtures/files/request.json")).read
   end
 
-  test "creates campuses" do
-    assert_difference "Campus.count", 14 do
-      JsonToEvents.call(@raw_json)
-    end
-  end
+  # test "creates campuses" do
```

Comment:
Did these tests stop passing?

Reply:
They didnt, it was to view the call from the DataRequestTest. Forgot to uncomment.